api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -425,16 +425,22 @@ type FelixConfigurationSpec struct {
 	// RouteTableRange specifies the indices of the route tables that Calico should use.
 	RouteTableRange *RouteTableRange `json:"routeTableRange,omitempty" validate:"omitempty"`
 
-	// WireguardEnabled controls whether Wireguard is enabled. [Default: false]
+	// WireguardEnabled controls whether Wireguard is enabled for IPv4. [Default: false]
 	WireguardEnabled *bool `json:"wireguardEnabled,omitempty"`
+	// WireguardEnabledV6 controls whether Wireguard is enabled for IPv6. [Default: false]
```

Comment:
@caseydavenport good point. After we decided to use a different wireguard interface for IPv6, both axes did seem to be bundled together (we'll have an IPv4 intf for IPv4 traffic and an IPv6 intf for IPv6 traffic). So in short in does mean both. I couldn't think of a use case that would justify mixing IPv4 and IPv6 wireguard traffic (as in encapsulating IPv4 traffic over the IPv6 intf, or vice versa), do you think that should be looked into? Otherwise, I think I can just make this comment clearer, what do you think?

Reply:
Yep, I think just cleaning up the comment is fine. The more important thing is that the comment (which eventually ends up as user-facing docs) explains the behavior precisely.

Reply:
I did try to clarify both IPv4 and IPv6 comments, please let me know if they're good now